src/lib.rs

```diff
@@ -22,6 +22,23 @@
 )]
 #![doc(html_root_url = "https://docs.rs/curve25519-dalek/4.0.0-pre.2")]
 #![doc = include_str!("../README.md")]
+//------------------------------------------------------------------------
+// curve25519-dalek arithmetric overrides
+//------------------------------------------------------------------------
+#![cfg_attr(
```

Comment:
@tarcieri this has the global default and override mechanism

Reply:
Aha!

Reply:
Oh you're right for some reason this doesn't set the cfg() here :see_no_evil: